tests/db_functions/tests.py

```diff
@@ -312,6 +312,37 @@ def test_least_decimal_filter(self):
             [obj],
         )
 
+    def test_abs(self):
```

Comment:
Can I suggest adding these tests to `tests/db_functions/test_math.py`?

(We ought to rearrange all the other database function tests to reflect the modules they are defined in.)

Reply:
Should I do this in this PR? Or leave it like this and wait for a separate PR to rearrange all the functions?